src/EnergyPlus/Psychrometrics.cc

```diff
@@ -734,29 +741,34 @@ namespace Psychrometrics {
 
         // If below -100C,set value of Pressure corresponding to Saturation Temperature of -100C.
         if (Tkel < 173.15) {
-            Pascal = 0.0017;
+            Pascal = 0.001405102123874164;
 
             // If below freezing, calculate saturation pressure over ice.
-        } else if (Tkel < DataGlobalConstants::KelvinConv) { // Tkel >= 173.15
-            Real64 constexpr C1(-5674.5359);                 // Coefficient for TKel < KelvinConvK
-            Real64 constexpr C2(6.3925247);                  // Coefficient for TKel < KelvinConvK
-            Real64 constexpr C3(-0.9677843e-2);              // Coefficient for TKel < KelvinConvK
-            Real64 constexpr C4(0.62215701e-6);              // Coefficient for TKel < KelvinConvK
-            Real64 constexpr C5(0.20747825e-8);              // Coefficient for TKel < KelvinConvK
-            Real64 constexpr C6(-0.9484024e-12);             // Coefficient for TKel < KelvinConvK
-            Real64 constexpr C7(4.1635019);                  // Coefficient for TKel < KelvinConvK
+        } else if (Tkel < DataGlobalConstants::TriplePointOfWaterTempKelvin) { // Tkel >= 173.15, Tkel < 273.16 (0.01°C)
+            Real64 constexpr C1(-5674.5359);
+            Real64 constexpr C2(6.3925247);
+            Real64 constexpr C3(-0.9677843e-2);
+            Real64 constexpr C4(0.62215701e-6);
+            Real64 constexpr C5(0.20747825e-8);
+            Real64 constexpr C6(-0.9484024e-12);
+            Real64 constexpr C7(4.1635019);
             Pascal = std::exp(C1 / Tkel + C2 + Tkel * (C3 + Tkel * (C4 + Tkel * (C5 + C6 * Tkel))) + C7 * std::log(Tkel));
 
             // If above freezing, calculate saturation pressure over liquid water.
-        } else if (Tkel <= 473.15) { // Tkel >= 173.15 // Tkel >= KelvinConv
+        } else if (Tkel <= 473.15) { // Tkel >= 173.15 // Tkel >= TriplePointOfWaterTempKelvin
```

Comment:
The else if above is Tkel < Real64 constexpr TriplePointOfWaterTempKelvin = 273.16; and this is <= 473.15 ... oh yeah, this is the top end so comment is wrong.

Reply:
These comments are a little confusing but a second looks shows they say what they said before with updates to what was changed. I'd say this is OK.

Reply:
I think the comments provide little value overall here, but there were pre-existing. and they should be correct.